how-to-create-typo-variable-fixer/composer.lock

```diff
@@ -0,0 +1,4514 @@
+{
+    "_readme": [
```

Comment:
Adding `coposer.lock` would bring only maintenance overhead. The `composer.json` is good enough here

Reply:
removed.